src/components/Card/CardStats.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from "react";
+import styled from "styled-components";
+import { Heading1, Text } from "../../typography";
+
+export interface CardStatsDetails {
+  label: string;
+  value: string;
+}
+
+export interface CardStatsProps {
+  details: CardStatsDetails[];
```

Comment:
You're saying that `details` is required here, but in line 15 you are treating it as if it was optional.

TypeScript will guarantee that `details` will exist, so we don't need the elvis operator here https://github.com/Major-Eth/yearn-web-lib/pull/27/files#r822211596

Reply:
This was removed with [this](https://github.com/Major-Eth/yearn-web-lib/pull/27#discussion_r822211596) `Commit Suggestion`